src/app/api/quality_matrix/quality_matrix.py

```diff
@@ -116,16 +110,22 @@ def missing_fields_ratio(value: dict, total_count: int) -> float:
 
 
 def missing_fields(
-    value: dict, total_count: int, replication_source: str
+    value: dict, total_count: int, search_keyword: str
 ) -> dict[str, float]:
-    return {replication_source: missing_fields_ratio(value, total_count)}
+    return {search_keyword: missing_fields_ratio(value, total_count)}
 
 
-async def stored_in_timeline(data: QUALITY_MATRIX_RETURN_TYPE, database: Database):
+async def stored_in_timeline(
```

Comment:
rename to `store_in_timeline`? The function inserts something, but it ready like a "check if something is in the database"

Reply:
Ay, good point. Changed.